assets/webauthn.js

```diff
@@ -13,17 +14,19 @@ document.addEventListener('DOMContentLoaded', function(){
 
     let attResp;
     try {
-      attResp = await startAttestation(await resp.json());
+      const jsonResponse = await resp.json();
+      if (jsonResponse.status === 500) {
```

Comment:
Apologies if I don't fully understand this. Does `.status` exist on jsonResponse? Or should it be on `resp`?

It might also be better to capture a wider array of status codes. You could use `resp.ok`. I think we'd want to also handle other 4xx and 5xx codes?

Reply:
`status` exists on the JSON response. It's coming from curveball/problem:
https://github.com/curveball/a12n-server/blob/master/src/main-mw.ts#L19
https://github.com/curveball/problem/blob/master/src/index.ts#L69

I'll make the error handling more robust.

Reply:
No longer using the `status` field and just doing `resp.ok` as suggested since it handles all the 5XX and 4XX codes. I also added some error handling if the fetch request itself fails.